Rubberduck.Parsing/Inspections/Resources/InspectionsUI.resx

```diff
@@ -848,6 +824,21 @@ If the parameter can be null, ignore this inspection result; passing a null valu
   <data name="EmptyWhileWendBlockInspectionName" xml:space="preserve">
     <value>Empty While-Wend block</value>
   </data>
+  <data name="EmptyConditionBlockInspectionsMeta" xml:space="preserve">
+    <value>A condition block should have executable statements. An empty block serves no purpose and can be confusing for the maintainer. Was this empty execution path originally intended to do something? Clear code only contains executable paths.</value>
+  </data>
+  <data name="EmptyConditionBlockInspectionsName" xml:space="preserve">
+    <value>Empty condition (If or Else)  block</value>
+  </data>
+  <data name="EmptyConditionBlockInspectionsQuickFix" xml:space="preserve">
+    <value>Remove the empty block. ***Note: Empty If necessitates applying 'Not' to the check ensuring the else block was inverted.</value>
```

Comment:
Description is too long. These populate a tiny little dropdown menu! Also I'm not sure what the "note" part means.